### ISSUE TITLE
Placing block problem

### ISSUE THREAD

**Issue by Adam1609** [LegendOfMCPE/EssentialsPE#220]
When I installed GrabBag, everything is fine, but after i installed esssentialsPe, something happened
.
I cant.place any block, even throwing potion, please help me.

**Comment by Tolo0** [comment]
Try removing grab bag

**Comment by PEMapModder** [comment]
@Tolo0 this is not resolve this issue. Please, do not provide suggestions that do not help solving the issue but only trying to run away from the issue.

**Comment by Tolo0** [comment]
Sorry. I just thought maybe grab bag was having conflict with EssentialsPE. I was just suggesting. To try it. Then if that didn't work. I would look into it more. I'm not trying to be a dick. Just trying to help.

**Comment by Adam1609** [comment]
I have essentialsPE latest 2.0.0 but placing block: unknown command

**Comment by Skull3x** [comment]
Yeah, unknown command appears and cancels block place for me

**Comment by Skull3x** [comment]
It's Power tool btw

**Comment by Skull3x** [comment]
How else would it say that, power tool adds commands to items

**Comment by Adam1609** [comment]
How to remove it?

**Comment by Tolo0** [comment]
Isn't it just. For example /powertool time set day. Then touch the floor. Then the item ur holding can set the time to day.
To remove it. Just do /powertool. While holding the item.

**Comment by Adam1609** [comment]
I really believe that powertool can be fix next time, guest cant do anything

**Comment by Adam1609** [comment]
I placed a block and it said: Unknown command: Type /help for a list of command
Somebody tell me....

**Comment by alex2534alex** [comment]
I also had that problem. I haven't any plugin which is installed. I installed command tracker plugin. When I place any block, it's show me "Unknown command: Type /help for a list of command". Then, I check commands log, but there is nothing...... Then, when some player enter to game, it's show me "Warning: Invalid argument supplied for foreach() in /SERVER/plugins/ESS_PE_SRC/SRC/EssentialsPE/Tasks/GeoLocation.php on line 32". This message isn't showing in log, only in console.
Anddd, can u say me how many mistakes I made? 3:
